env

```diff
@@ -0,0 +1,75 @@
+#See more informations here : https://doc.getlago.com/guide/self-hosted/docker
+
+LAGO_VERSION=v1.13.1
+
+# PostgreSQL Configuration
+POSTGRES_DB=lago
+POSTGRES_USER=lago
+POSTGRES_PASSWORD=changeme
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
+POSTGRES_SCHEMA=public
+PGDATA=/data/postgres
+
+# Redis Configuration
+REDIS_HOST=redis
+REDIS_PORT=6379
+REDIS_PASSWORD=
+
+# Application Ports
+API_PORT=3000
+FRONT_PORT=80
+
+
+# Lago API Configuration
+DOMAIN=yourdomain.told
```

Comment:
this is a whole configuration and not only `API` 
maybe it's better to add a global config part, wdyt?

Reply:
also can we rename it `LAGO_DOMAIN` ?
I think we'll use the same for our helm chart to ease the ingress configuration!